lib/src/widgets/google_places_autocomplete_text_field.dart

```diff
@@ -317,20 +326,27 @@ class _GooglePlacesAutoCompleteTextFormFieldState
       return;
     }
 
-    final result = await _api.getSuggestionsForInput(
-      input: text,
-      googleAPIKey: widget.googleAPIKey,
-      countries: widget.countries ?? [],
-      sessionToken: widget.sessionToken,
-      proxyUrl: widget.proxyURL ?? "",
-    );
-
-    if (result == null) return;
-    final predictions = result.predictions;
-    if (predictions == null || predictions.isEmpty) return;
+    try {
+      final result = await _api.getSuggestionsForInput(
+        input: text,
+        googleAPIKey: widget.googleAPIKey,
+        countries: widget.countries ?? [],
+        sessionToken: widget.sessionToken,
+        proxyUrl: widget.proxyURL ?? "",
+        languageCode: widget.languageCode,
+      );
+      if (result == null) return;
+      final predictions = result.predictions;
+      if (predictions == null || predictions.isEmpty) return;
 
-    allPredictions.clear();
-    allPredictions.addAll(predictions);
+      allPredictions.clear();
+      allPredictions.addAll(predictions);
+    } catch (e) {
+      debugPrint('getLocation: ${e.toString()}');
+      if (widget.onError != null) {
```

Comment:
no need for the if statement when you're using `widget.onError?.call()`

Reply:
I have removed the if